src/dashboard/Push/PushAudiencesIndex.react.js

```diff
@@ -245,7 +245,7 @@ class PushAudiencesIndex extends DashboardView {
       }}>
     </FormModal>
 
-    if (typeof(data) !== undefined) {
+    if (typeof data !== 'undefined') {
```

Comment:
Why so complicated? What is the difference between `typeof data !== 'undefined'`and `data !== undefined`?

Reply:
In this case there would not be any difference as the `data` variable exists.